pkg/crc/machine/machine.go

```diff
@@ -517,12 +517,12 @@ func (client *client) Status() (*ClusterStatusResult, error) {
 	libMachineAPIClient, cleanup, err := createLibMachineClient(false)
 	defer cleanup()
 	if err != nil {
-		return statusError("Cannot initialize libmachine", err)
+		return nil, err
```

Comment:
This does not feel very consistent, `Simplify machine.Client Stop` directly removed `stopError` and inlined it instead:
`return state.None, errors.Wrap(err, "Cannot initialize logging")`

With this commit, why do we remove the `error.Wrap()` from console/status/...? I'd just keep it as this makes it easier to know where in the code the error comes from. And why do we treat `startError()` differently from `stopError()`?

Reply:
I reworked this. It should be correct now. I removed the function error function at the same type I touch the related machine function.